functions.sh

```diff
@@ -3296,12 +3296,12 @@ report_statistic() {
   if [ "$1" -a "$2" -a "$3" -a "$4" -a "$5" ]; then
     REPORTSRV="$1"
 
-    STANDARDIMAGE="`ls -1 "$IMAGESPATH" |grep "$2"`"
+    STANDARDIMAGE="$(ls -1 "$IMAGESPATH" |grep "$2")"
```

Comment:
What is $2 here? a whole path? why the call with "basename" AND "$IMAGESPATH"? If "$2" is just a string containing "/" and you want everything after the last "/", a "${2##*/}" is sufficent.